src/lib/holocene/nav-container.svelte

```diff
@@ -0,0 +1,110 @@
+<script lang="ts">
+  import Icon from '$lib/holocene/icon/index.svelte';
+
+  import { navOpen } from '$lib/stores/nav-open';
```

Comment:
🆗 to import stores from `src/lib/stores` in holocene? Or should `src/lib/holocene` have its own `stores` sub-directory?

Reply:
I was pondering this as well... I think in this case we should supply the store to holocene so cloud doesn't care or know about the store. But for pages and other things we want to allow the import of stores and add them to pages